src/components/XPubTable/columns.tsx

```diff
@@ -13,21 +14,37 @@ export const columns: ColumnDef<XpubsColumns>[] = [
     accessorKey: 'id',
     header: ({ column }) => {
       return (
-        <Button variant="ghost" onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}>
-          Id
-          <ArrowUpDown className="ml-2 h-4 w-4" />
-        </Button>
+        <Link
+          from={'/xpub'}
+          search={{
+            order_by_field: 'id',
+            sort_direction: column.getIsSorted() === false ? 'asc' : (column.getIsSorted() as string),
```

Comment:
Why is `as string` necessary?

Reply:
Because it can have three states. False, asc and desc. If it's not false then it should be a string. `sort_direction` should be a string.

Reply:
Added a func